### PR TITLE
CLI Code snippets

### PR DESCRIPTION
**Pull request by calebkiage**
Fix bug with missing query parameter values

### CODE REVIEW

File: pipelines/unused-snippets.yml [hunk 1]
```diff
@@ -5,21 +5,21 @@
...
 trigger: none
 pr: none
-schedules:
-  - cron: "30 12 * * 3"
-    displayName: Weekly Wednesday unused snippets removal
-    branches:
-      include:
-      - master
-    always: true
+# schedules:
```
Comment (fey101) [inline pipelines/unused-snippets.yml:8]: Why are we commenting this out? We don't want to have unreferenced snippets removed/cleaned up for this branch?

File: pipelines/snippets.yml [hunk 2]
```diff
@@ -5,21 +5,21 @@
 
 trigger: none
 pr: none
```
Comment (fey101) [inline pipelines/snippets.yml:7]: Why are we disabling scheduled runs for snippet generation?

File: CodeSnippetsReflection.OpenAPI/LanguageGenerators/PowerShellGenerator.cs [hunk 3]
```diff
@@ -320,14 +422,15 @@ private static void WriteJsonArrayValue(StringBuilder payloadSB, JsonElement val
...
         {
-            if (schema == default) return default;
-            return schema.Type switch
-            {
-                "integer" when schema.Format.Equals("int64") => $"{value.GetInt64()}",
-                _ when schema.Format.Equals("float") => $"{value.GetDecimal()}",
-                _ when schema.Format.Equals("double") => $"{value.GetDouble()}",
-                _ => value.GetInt32().ToString(),
-            };
+            //if (schema == default) return default;
```
Comment (fey101) [inline CodeSnippetsReflection.OpenAPI/LanguageGenerators/PowerShellGenerator.cs:425]: Code smell. Though just noticed the PR is still draft.

**Comment by timayabi2020** [comment]
## Fixes
#1621 
#1622
#1623 
#1624

**Comment by calebkiage** [comment]
@timayabi2020, could you please use separate branches for each of the feature fixes (#1621
, #1622, #1623, #1624) then open a PR against dev? The work on API doctor should be separated from the fixes since we want to merge the fixes as they become available.

**Comment by timayabi2020** [comment]
@calebkiage Yes, I had actually started working on the targeted branches for those issues, with the aim of merging to this branch, but I wasn't able to. I have been using this branch to run the snippet generation pipeline for cli and testing those snippets. I will close it once I'm done with the tests.

**Comment by timayabi2020** [comment]
Closing this PR to work on fixes with targeted branches that will be merged to dev

**Pull request closed without merging**